codice/ddf
# Cdm Set Max File Size

#### What does this PR do?
Prevents CDM from ingesting files where the file size exceeds 1 GB. Max file size can be configured in **custom.system.properties**

#### Who is reviewing it? 
@mdang8 @aaronilovici 
@beyelerb 


#### Ask 2 committers to review/merge the PR and tag them here.
@clockard 

#### How should this be tested?
1. Set up an in place directory monitor
2. In **etc/custom.system.properties**, set **org.codice.ddf.catalog.content.monitor.maxFileSizeBytes** to a max file size (in bytes).
2. Ingest a file smaller than the set maximum and a file larger than it. Verify that the smaller ingests and the larger does not.

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
